From c9d1d028033166d456465aec5c6da01fa1bb7248 Mon Sep 17 00:00:00 2001
From: Jiawei Huang <hjiawei@gmail.com>
Date: Mon, 19 Apr 2021 22:56:51 -0700
Subject: [PATCH] Alwasy get release version from conf.py

---
 py/makeversionhdr.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/py/makeversionhdr.py b/py/makeversionhdr.py
index 2f4bc91..c7e647c 100644
--- a/py/makeversionhdr.py
+++ b/py/makeversionhdr.py
@@ -74,9 +74,7 @@ def get_version_info_from_docs_conf():
 
 def make_version_header(filename):
     # Get version info using git, with fallback to docs/conf.py
-    info = get_version_info_from_git()
-    if info is None:
-        info = get_version_info_from_docs_conf()
+    info = get_version_info_from_docs_conf()
 
     git_tag, git_hash = info
 
-- 
2.30.2

